lib/WebpackOptionsDefaulter.js

```diff
@@ -270,7 +270,9 @@ class WebpackOptionsDefaulter extends OptionsDefaulter {
 		this.set("optimization.splitChunks.maxInitialRequests", "make", options => {
 			return isProductionLikeMode(options) ? 4 : Infinity;
 		});
-		this.set("optimization.splitChunks.name", true);
+		this.set("optimization.splitChunks.name", "make", options => {
+			return isProductionLikeMode(options);
```

Comment:
I think the test should be inverted. According to Tobias, we want `false` in production